docs/scripts/curate_soma_experiment.py

```diff
@@ -0,0 +1,42 @@
+import lamindb as ln
+import bionty as bt
+import tiledbsoma as soma
+import tiledbsoma.io
+
+adata = ln.core.datasets.small_dataset1(otype="AnnData")
+tiledbsoma.io.from_anndata("curate.tiledbsoma", adata, measurement_name="RNA")
```

Comment:
This is a nitpick, but I don't find the choice for the folder name very intuitive. We're typically using `my_dataset.tiledbsoma` or something along these lines. I tried to standardize this with the idea of showcasing that "with lamindb you can store the same dataset in different formats".